apps/meteor/app/threads/server/methods/followMessage.js

```diff
@@ -31,6 +32,9 @@ Meteor.methods({
 			throw new Meteor.Error('error-not-allowed', 'not-allowed', { method: 'followMessage' });
 		}
 
+		const isFollowed = true;
+		Promise.await(Apps.triggerEvent(AppEvents.IPostMessageFollowed, message, Meteor.user(), isFollowed));
```

Comment:
This got me thinking, for the event to be _post_ message followed, shouldn't it be ran _after_ the `follow` function all?